spec/models/job_spec.rb

```diff
@@ -54,4 +54,30 @@
       expect(job.author).to eq author
     end
   end
+
+  describe ".search" do
+    it "returns a job when the query matches its title" do
+      matching_title_job = create(:job, :published, title: "Fullstack developer")
+      expect(Job.search("fullstack")).to include matching_title_job
+    end
```

Comment:
I know the specs for that class are not very consistent. But in an effort to bring more clarity to those, I suggest that you use `subject` and `let` for your tests instead of declaring variables and call the method directly in the expectation.